custom_components/tuya_local/devices/catit_pixi_smart_fountain.yaml

```diff
@@ -90,3 +90,46 @@ entities:
         type: integer
         name: sensor
         unit: s
+
+  - entity: sensor
+    name: Water Lack
+    dps:
+      - id: 101
+        type: boolean
+        name: sensor
+
+  - entity: switch
+    name: Eco Mode
```

Comment:
Follow HA naming convention: only first word of an entity name should be capitalised.